app/views/api_docs/pages/release_notes.md

```diff
@@ -1,3 +1,9 @@
+### 19th November 2020
+
+- The `Qualification` object now supports multiple types of English GCSEs (eg. English Language, English Studies Double 
+Award). Candidates may have multiple English GCSEs. Each GCSE is provided as a separate `Qualification`. The title of 
+the GCSE is given in the `subject` field. 
+
```

Comment:
I realize now that a consumer of the API can no longer identify the English GCSEs for certain. Since we have an "Other English GCSE" option with text input they could put in whatever and there's no clear way to show it as an English GCSE.

```                    {
                        "id": null,
                        "qualification_type": "gcse",
                        "non_uk_qualification_type": null,
                        "subject": "Cockney rhyming slang",
                        "grade": "A*",
                        "start_year": null,
                        "award_year": "1997",
                        "institution_details": null,
                        "awarding_body": null,
                        "equivalency_details": null,
                        "hesa_degtype": null,
                        "hesa_degsbj": null,
                        "hesa_degclss": null,
                        "hesa_degest": null,
                        "hesa_degctry": null,
                        "hesa_degstdt": null,
                        "hesa_degenddt": null
                    }
```

This isn't currently an issue elsewhere in the codebase, so we could potentially add a new field to the `Qualification` object if we wanted to mark the qualifications clearly as English or science etc.

Reply:
Sounds like a sensible mitigation if we need it. I'm not worried about this right now as we effectively guarantee that an application in the API will have English and Maths GCSEs, and I think providers will need to read these things anyway to evaluate them.